src/open_inwoner/openklant/forms.py

```diff
@@ -45,6 +47,12 @@ class ContactForm(Form):
         widget=forms.Textarea(attrs={"rows": "5"}),
         required=True,
     )
+    captcha = MathCaptchaField(
+        label=_("Beantwoord deze rekensom"),
```

Comment:
This sentence has to be in "actief taalgebruik" and it also has to be "B1 niveau" so this is the best I could come up with.